CreditManagement/views.py

```diff
@@ -24,7 +22,7 @@ class TransactionAddView(LoginRequiredMixin, View):
     template_name = "credit_management/transaction_add.html"
     context = {}
 
-    def get(self, request, association_name=None, *args, **kwargs):
+    def get(self, request, association_name=None):
```

Comment:
I've had some problems in instances where `*args` and `**kwargs` were left out for these request handling methods (get/post). If it works for you then fine, but maybe check it.